src/main/deltachat/chatlist.js

```diff
@@ -115,7 +119,8 @@ function getSmallChatById (chatId, list, i) {
     isVerified: chat.isVerified,
     isGroup: chat.isGroup,
     freshMessageCounter: chat.freshMessageCounter,
-    isArchiveLink: chat.isArchiveLink
+    isArchiveLink: chat.isArchiveLink,
+    selfInGroup: chat.selfInGroup
```

Comment:
I propose isSelfInGroup, just to be consistent with the two variables above. Naming is hard and i was just reminded about our talk about naming some days ago ^^

Reply:
hmm - I don't agree. isArchiveLink means this chat IS an archive link. selfInGroup means "you are in the group of this chat" not the chat IS selfInGroup...
naming issues... ;-)

Reply:
Fair :)